kmk/extensions/peg_rgb_matrix.py

```diff
@@ -1,19 +1,22 @@
 import neopixel
 
 from storage import getmount
-
 from kmk.extensions import Extension
```

Comment:
```suggestion
from kmk.extensions import Extension

```
Think it wants a newline